sql/pgsql/_debug_athletes_footballNcaa_2026.sql

```
@@ -0,0 +1,49 @@
+select * from public."AthleteSeason" limit 10;
+
+select * from public."AthletePosition" order by "Name";
```

SQLFluff (4.3.0)

[error] The 'order' keyword should always start a new line.

(LT14)